docker-compose.local.yml

```diff
@@ -1,33 +1,32 @@
-
 services:
-  postgres: 
+  postgres:
     image: postgres
     container_name: postgress_db
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: password
       POSTGRES_DB: mydb
     ports:
-      - "5432:5432" # Data base port
+      - '5433:5432' # Data base port
     volumes:
       - pgdata:/var/lib/postgresql/data
   pgadmin:
-      image: dpage/pgadmin4
-      container_name: pgadmin
-      environment:
-        PGADMIN_DEFAULT_EMAIL: admin@admin.com
-        PGADMIN_DEFAULT_PASSWORD: admin
-      ports:
-        - "8080:80"
-      depends_on:
-        - postgres
+    image: dpage/pgadmin4
+    container_name: pgadmin
+    environment:
```

Comment:
these are mostly formatting changes. i think we can just revert this file entirely right?